package.json

```diff
@@ -2,7 +2,6 @@
 	"name": "wallets",
 	"private": true,
 	"scripts": {
-		"postinstall": "pnpm deps:check",
```

Comment:
?

Reply:
I disable it every time as when I want to update a dep I have to do it in 5 places and running `pnpm add expo@52` for example does not work as it does not apply the changes to the package.json because it errors..

I will enable it again, but I find it too strict. It should error in CI, not local dev.

Reply:
I think the new pnpm feature to specify catalogs can help here.

Consistent versions is a good thing. Doing it only in ci is fine (if you can make a PR?)